scripts/sync-bottle-hashes.sh

```diff
@@ -30,7 +30,7 @@ fi
 while : ; do
     git fetch --all
     git reset --hard origin/"$branch_name"
-    ./scripts/bottle-hashes.sh .
+    ./scripts/bottle-hashes.sh "./$2"
```

Comment:
Could you explain why this was changed? Seems like it expects bottles to be in `<macOS name>` directories, but I can't find whether bottles are actually moved to these directories